src/main/java/practice/StreamPractice.java

```diff
@@ -13,17 +16,32 @@
      * If there is no needed data throw RuntimeException with message
      * "Can't get min value from list: < Here is our input 'numbers' >"
      */
+
     public int findMinEvenNumber(List<String> numbers) {
-        return 0;
+        String messageRE = "Can't get min value from list: < Here is our input 'numbers' >";
+        return numbers
+                .stream()
+                .map(x -> x.split(","))
+                .flatMap(Arrays::stream)
+                .map(Integer::parseInt)
+                .filter(x -> x % 2 == 0)
+                .min(Integer::compare)
+                .orElseThrow(() -> new RuntimeException(messageRE));
```

Comment:
```suggestion
        return numbers.stream()
		.flatMap(n -> Arrays.stream(n.split(",")))
                .map(Integer::parseInt)
                .filter(x -> x % 2 == 0)
                .min(Integer::compare)
                .orElseThrow(() -> new NoSuchElementException("Can't get min value from list " + numbers));
```

Операции map и flatMap мы можем объеденить в одну flatMap. 
Мне кажется NoSuchElementException более информативна в этом случае
< Here is our input 'numbers' > - не очень поняла как это используется